internal/cmd/grpc.go

```diff
@@ -219,7 +219,13 @@ func NewGRPCServer(
 			return status.Errorf(codes.Internal, "%v", p)
 		})),
 		grpc_ctxtags.UnaryServerInterceptor(),
-		grpc_zap.UnaryServerInterceptor(logger),
+		grpc_zap.UnaryServerInterceptor(logger, grpc_zap.WithDecider(func(methodFullName string, err error) bool {
+			// will not log gRPC calls if it was a call to healthcheck and no error was raised
+			if err == nil && methodFullName == "grpc.health.v1.Health/Check" {
```

Comment:
Something is not right. I still see logs. Should it be `/grpc.health.v1.Health/Check`?

Reply:
ah you are right! good find. I should have tested better